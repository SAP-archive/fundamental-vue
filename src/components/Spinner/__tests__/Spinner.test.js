import { mount } from "@vue/test-utils";
import { Spinner } from "../";

describe("Spinner", () => {
  const spinner = mount(Spinner, {
    propsData: {
      ariaLabel: "Label"
    }
  });
  test("renders correctly", () => {
    expect(spinner.element).toMatchSnapshot();
  });
  test("renders correct props", () => {
    expect(spinner.attributes("aria-label")).toEqual("Label");
  });
});
