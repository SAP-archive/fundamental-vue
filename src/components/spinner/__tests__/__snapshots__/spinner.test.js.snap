// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner renders correctly 1`] = `
<div
  aria-hidden="false"
  aria-label="Label"
  class="fd-spinner"
>
  <div
    class="fd-spinner__body"
  />
</div>
`;
